src/test/java/utilities/TestListeners.java

```diff
@@ -0,0 +1,60 @@
+package utilities;
+
+import com.codeborne.selenide.WebDriverRunner;
+import io.qameta.allure.Attachment;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.testng.ITestContext;
+import org.testng.ITestListener;
+import org.testng.ITestResult;
+
+public class TestListeners implements ITestListener {
+
+    WebDriver driver;
+    WebDriverRunner webDriverRunner;
+
+    @Override
+    public void onTestStart(ITestResult iTestResult) {
+
+    }
+
+    @Attachment
+    @Override
+    public void onTestSuccess(ITestResult iTestResult) {
+
+        System.out.println("Test ".concat(iTestResult.getName()).concat(" started"));
```

Comment:
Seems like this message about test start should be moved to 'onTestStart', also it'd be better to use standard logging instead of printing to console